src/ui/menu/menuView.tsx

```diff
@@ -174,6 +174,11 @@ function renderLinks(user?: Session) {
           <span data-icon="4" />{i18n('studyMenu')}
         </li> : null
       }
+      {online ?
+        <li className="side_link" data-route="/coord">
+          <span data-icon="4" />{i18n('coordinates')}
+        </li> : null
+      }
```

Comment:
Does the app need to be online for coordinates training?

Reply:
This feature should be available offline yes. It's just that when offline we don't send the score to the server.

Reply:
@jas14 no it doesn't. It was a mistake, I forgot to remove it after fiddling with the code at first.